src/helpers/url.js

```diff
@@ -32,13 +33,26 @@ const getSearchParam = (name) => {
 	return decodeURI(results[1]) || ''
 }
 
+const getCallbackBaseUrl = () => {
+	const callbackUrl = Config.get('wopi_callback_url') || loadState('richdocuments', 'wopi_callback_url', '')
+	return callbackUrl || window.location.protocol + '//' + window.location.host + getRootUrl()
+}
+
+const getWopiSrc = (fileId) => {
+	// WOPISrc - URL that loolwsd will access (ie. pointing to ownCloud)
```

Comment:
:upside_down_face: for the sake of backport :cocktail:

Reply:
🙈 Updated

Reply:
i thought it is intentionally kept to not have differences with higher branches